metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -332,8 +334,12 @@ def __init__(self, config, ctx):
         self.partition_info: Dict[str, str] = dict()
         atexit.register(cleanup, config)
 
-    def get_db_name(self, inspector: Inspector = None) -> str:
-        if self.config.project_id:
+    def get_db_name(
+        self, inspector: Inspector = None, for_sql_queries: bool = True
```

Comment:
Maybe some comment would be nice about the `for_sql_queries` parameter

Reply:
added